cordova-plugin-mobilespec-tests/tests/datauri.tests.js

```diff
@@ -27,6 +27,7 @@ exports.defineAutoTests = function () {
 
         function onMessage(done, msg) {
             if (msg.data == 'foo') {
+                expect('Suppress SPEC HAS NO EXPECTATIONS').toBeDefined();
```

Comment:
it's not my code...I just fixed the warning :-) I think the test has no expectations because it just needs to wait for the async() to finish. Also, I think this warning was fixed in a more modern jasmine version.